lib/screens/explore_screens/regions_screens/edit_region/edit_region_image.dart

```diff
@@ -67,7 +62,7 @@ class EditRegionImage extends StatelessWidget {
                       title: context.loc.createRegionAddBackGroundImageBoxTitle,
                       onTap: () => BlocProvider.of<EditRegionBloc>(context).add(const OnPickImage())),
                   const SizedBox(height: 10),
-                  if (state.file != null)
+                  if (state.file != null && !state.isSaveChangesButtonLoading)
```

Comment:
Hide replace button on saving image.

Reply:
I like this, but lest remove the logic from the UI and disable the button instead of hiding it.

Reply:
fixed